roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -2289,7 +2289,17 @@ RooDataSet *RooAbsPdf::generate(const RooArgSet &whatVars, const RooDataSet& pro
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+
+std::string RooAbsPdf::translate(RooFit::Detail::CodeSquashContext &ctx) const
```

Comment:
I noticed that this implementation ended up in the wrong class. The RooAbsPdf in general should not override translate. This implementation with the division you have here should go into the `RooNormalizedPdf` class.

And then you also don't need to "guess" where the function and integral is in the list of servers. The implementation will look like this:
```c++
std::string RooNormalizedPdf::translate(RooFit::Detail::CodeSquashContext &ctx) const
{
   // For now just return function/normalization integral.
   ctx.addResult(this, ctx.getResult(&_pdf.arg()) + "/" + ctx.getResult(&_normIntegral.arg()));

   return "";
}
```
And if you already don't take the strings by `const &`, at least put them inline in the `addResult()` call so that we are sure to deal with temporaries.